src/covariance.jl

```diff
@@ -16,11 +16,52 @@ Evaluate the covariance at objects `x₁` and `x₁`.
 """
 (cov::Covariance)(x₁, x₂) = sill(cov.γ) - cov.γ(x₁, x₂)
 
+"""
+    isstationary(cov)
+
+Check if covariance `cov` possesses the 2nd-order stationary property.
+"""
+isstationary(cov::Covariance) = isstationary(typeof(cov.γ))
+
+"""
+    isisotropic(cov)
+
+Tells whether or not covariance `cov` is isotropic.
+"""
+isisotropic(cov::Covariance) = isisotropic(metricball(cov.γ))
+
+for fun in (:sill, :nugget, :metricball)
+  eval(quote
+    """
+        $($fun)(cov)
+
+    Returns the $($fun) of the covariance `cov`.
+    """
+    $fun(cov::Covariance) = $fun(cov.γ)
+  end)
+end
```

Comment:
Let's simply copy/paste the three functions. We can use metaprogramming if the number of function increases in the future.